src/platformsupport/input/evdevtouch/qevdevtouchhandler.cpp

```diff
@@ -213,6 +216,16 @@ QEvdevTouchScreenHandler::QEvdevTouchScreenHandler(const QString &device, const
             invertx = true;
         } else if (args.at(i) == QLatin1String("inverty")) {
             inverty = true;
+        } else if (QStringLiteral("/dev/%1").arg(args.at(i)).startsWith(device)) {
+            QString option = args.at(i).section(QLatin1Char('/'), -1);
+            if (option.startsWith(QLatin1String("pencil="))) {
+                QString optionValue = option.section(QLatin1Char('='), 1, 1);
```

Comment:
This takes some staring to figure out... an explanatory comment on what arg is expected to hold could be useful for a casual reader.